code/script.js

```diff
@@ -0,0 +1,115 @@
+const username = 'NabeelMansour'
+const container = document.getElementById('container')
+const header = document.getElementById('header')
+const technigoRepos = document.getElementById('technigoRepos')
+
+
+// DOM selectors
+const userName = document.getElementById('username')
+const project = document.getElementById('projects')
+const chart = document.getElementById('chart');
+
+const options = {
+    method: 'GET',
+    headers: {
+        Authorization: `token ${token}`
+    }
+}
+
+
+
+
+// API'S
+const USER_API = `https://api.github.com/users/${username}`
+const API_REPOS = `https://api.github.com/users/${username}/repos`
+
+
+
+// fetch for the user info
+
+fetch(USER_API, )
+.then(res => res.json())
+.then(data => {
+    console.log(data)
+    header.innerHTML += `
+    <div class="header-text">
+      <img src="${data.avatar_url}" class="image" />
+      <p>${data.name}</p>
+      <h1><span>${data.login}</span></h1>
+      <h2>GitHub Tracker</h2>
+      <p>${data.bio}</p>
+    </div>
+    `
+    
+  }) 
+  
+// fetch the users reposetories
+
+fetch(API_REPOS)
+    .then(res => res.json())
+     .then(data => {
+        // console.log(data)
+        const technigoRepositories = data.filter(repo => repo.name.includes('project-') && repo.fork)
+
+         technigoRepositories.forEach((repo) => {
+          container.innerHTML += `
+          <div class="technigo-repos" id="technigoRepos">
+            <h2 id="repoName">${repo.name}</h2>
+            <h3 id="description">${repo.description}</h3>
```

Comment:
It seems like there is no description so the page throw null. Better to remove this.